src/pages/open-source-for-devs.md

```diff
@@ -0,0 +1,32 @@
+
+## More than Software
+
+"Open source" is commonly associated with software, but other types of technology, such as hardware, also adopted the term. Additionally, other fields have embraced open source principles, such as transparency, collaboration, community, and release early and often to help build better communities and initiatives. For example, [All In](https://allinopensource.org/) is a community dedicated to advancing diversity, equity, and inclusion within open source. [Demetris Cheatham](https://youtu.be/UwbrgUGlqH4) built All In on open source practices with the goal of “open sourcing” diversity or sharing diversity practices among open source communities and Diversity, Equity, and Inclusion Practitioners.
+
+## Why would a project choose to be open source
+
+It may seem odd that a company, product, or tool would choose to be open source. After all, wouldn’t you want to keep your code private so that no one can take your ideas? People open source their projects for many reasons. Here are a few common reasons why someone might open source a project:
+
+### Improving company brand
+
+Companies like Google, Facebook, and Goldman Sachs are heavily involved in open source. Many companies recognize that open sourcing a project improves the organization's reputation through:
+
+- **Marketing** - Users who love the product will feel more compelled and excited to help make the product better. Contributing can fuel users’ passion even more, and they may start encouraging those in their network to use or contribute to our project. Some contributors may even talk about the project at conferences, on Twitch live streams, in blog posts, and in YouTube tutorials. That's free marketing to help the organization acquire customers and developers!
+- **Hiring** - Developers admire companies that are transparent about their technological growth and processes. By documenting the company’s journey and as contributors increase awareness of the product, businesses can increase their chances of attracting highly-skilled, community-driven, empathetic developers.
+- **Longevity** - With increased collaboration, an organization can gain access to developers with varied technological experiences. They can help keep the project up to date with the latest technological tools, and contributors can help add integrations faster. For example, before the rise of container orchestration, both Spotify and Google developed their own systems called Helios and Kubernetes. Google chose to open source Kubernetes, while Spotify kept Helios as an internal project. Unfortunately, Spotify didn't have enough capacity to continue the project internally. Over the past seven years, Kubernetes has been a top choice for container orchestration; even Spotify migrated its containerization to Kubernetes.
```

Comment:
Maybe you can put the reasons underneath the paragraph in line 16 to make the section more organized?